src/hooks/usePokemonQuery.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { useQuery } from './useQuery';
+import { MAX_CURRENT_POKEMON } from '../components/Pokedex/pokedexConstants';
+import { useHistory, useLocation } from '@docusaurus/router';
+
+const POKEMON_QUERY_PARAMETER_NAME = 'pokemon';
+
+const isValidPokemonDexId = (pokemonDexId) => pokemonDexId >= 0 && pokemonDexId <= MAX_CURRENT_POKEMON;
```

Comment:
Are you using this to search by monsno or ID? 

Pokemon ID is separate from dex ID, monsno would be what you would think of as DexID. 

Either way, 493 is too low of a max and the links will not be what the users expect, so an alternative form of expression is likely required to include monsno and formno.

Reply:
yea - i guess we will have to think of how we wanna do it